linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_finit/ansible/shared.yml

```diff
@@ -4,12 +4,18 @@
 # disruption = low
 # strategy = configure
 
-{{% if "ol" in product or 'rhel' in product %}}
+{{% if product in ['ol', 'rhel', 'ubuntu2004'] %}}
 {{% set auid_filters = "-F auid>=" ~ auid ~ " -F auid!=unset" %}}
 {{% else %}}
 {{% set auid_filters = "" %}}
 {{% endif %}}
 
+{{% if 'ubuntu2004' in product %}}
```

Comment:
please remove this entire block, the key name is not relevant, we prefer to stick with a common key